ibis/backends/base/__init__.py

```diff
@@ -135,6 +135,8 @@ class BaseBackend(abc.ABC):
 
     database_class = Database
     table_class: type[ops.DatabaseTable] = ops.DatabaseTable
+    con_str = None
```

Comment:
none of this can be here

it should be in the sql base

Reply:
Agreed, I'll move them to a better place.

Reply:
Moved to `__init__` constructors.